app/forms/itt_start_date_form.rb

```diff
@@ -1,6 +1,17 @@
 # frozen_string_literal: true
 
 class IttStartDateForm < MultiDateForm
+  def stash
+    form = CourseDetailsForm.new(trainee)
```

Comment:
We need this extra method to avoid running `valid?`. We save a subset of the fields sometimes and the form does not validate, which stops it from being stashed.